src/dotnet-restore/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "dotnet-restore",
+  "description": "Restore packages specified in project.json (project file)",
+  "version": "1.0.0-*",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dotnet/cli"
+  },
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "emitEntryPoint": true
+  },
+  "dependencies": {
```

Comment:
What is the problem we are warning for though?
Including additional references like this is a valid scenario if you wanted to override the stdlib libraries. 

Additionally, the same logic would be presumably used to resolve between mismatching versions in the dependency graph.